package.json

```diff
@@ -135,8 +135,9 @@
     "graph-types": "graphql-codegen --config ./graph-types.codegen.yaml",    
     "lingui:extract": "lingui extract",
     "lingui:compile": "lingui compile",
+    "lingui:branch:main": "git submodule set-branch --branch main src/locales/translations/",
     "typechain:build": "yarn run typechain --target ethers-v5 --out-dir src/typechain src/abi/*.json src/abi/**/*.json",
-    "preinstall": "[ ! -f src/locales/translations/.git ] && git submodule deinit --all -f; git submodule update --init --recursive --remote",
+    "preinstall": "git submodule update --init --recursive --remote",
```

Comment:
did you test this with a fresh build from scratch? I want to make sure we don't have that problem where if a user runs `yarn install` before `yarn lingui:branch:main` then they can never get the languages.

Reply:
Hi guys. Thanks for the feedback and the tips.
Yeah yarn `lingui:branch:main` just changes the branch in the git config, not in the actual repo filesystem `/src/locales/translations` (I made a fresh install after pushing to make sure).
The preinstall test that avoids refetching languages works properly (on my machine at least)

Reply:
so I just tested this branch on a fresh install. Everything works, but I have one question: 

1. assume I'm just a random new dev. I clone the repo & run `yarn install` per the README. the install command pulls in all of the translations & makes `translations.diff` a commit-able file. Is that the expected behavior?

Reply:
And then after committing the changes that `yarn install` brought in (here: https://github.com/OlympusDAO/olympus-frontend/pull/719/commits/81ad778b466602f11ba285070304b7db79fb8534)

I still have unstaged changes.
![image](https://user-images.githubusercontent.com/80423742/142768079-dc77bf4b-334b-4324-8ac8-82300718c7c8.png)

Reply:
I think it is expected albeit undesirable. I found something that may help there: https://stackoverflow.com/questions/3240881/git-can-i-suppress-listing-of-modified-content-dirty-submodule-entries-in-sta
I will try to look into it.

Reply:
cool, that stack overflow suggestion (adding `ignore = dirty` to `.gitmodules` definitely gets rid of the untracked changes issue.

Reply:
ok this commit https://github.com/OlympusDAO/olympus-frontend/pull/719/commits/3a301addfeee39bc65254c9a5250720e03d4a403 adds `ignore = dirty`